src/video_core/shader/node.h

```diff
@@ -464,6 +464,10 @@ class OperationNode final : public AmendNode {
         return operands.size();
     }
 
+    NodeBlock GetOperands() const {
```

Comment:
```suggestion
    const NodeBlock& GetOperands() const {
        return operands;
    }
```

So usages of this function don't churn std::vector allocations.